src/services/reporting/carSales.ts

```diff
@@ -39,10 +39,7 @@ const envelopeRequest = async (handler): Promise<WithValidationError<null>> => {
       result: null,
     }
   } catch (error) {
-    return handleValidationError({
-      error,
-      options: { swallowErrors: true },
-    })
+    return handleValidationError(error, { swallowErrors: true })
```

Comment:
@ufkaya I think I found the "real mistake". Before it was `handleValidationError({error, options :{ swallowErrors: true }})`
This would explain why e`rror.statusCode` was not enough and that I had to `error.error.statusCode`

Reply:
Nice! Good catch!